cmd/session-v8.go

```diff
@@ -375,6 +375,18 @@ func (s sessionV8) CloseAndDie() {
 	console.Fatalln("Session safely terminated. To resume session `mc session resume " + s.SessionID + "`")
 }
 
+func (s sessionV8) copyCloseAndDie(sessionFlag bool) {
+	if sessionFlag {
+		s.Close()
+		console.Fatalln("Command terminated safely. Run this command to resume copy again.")
```

Comment:
question: This would result in exit code being non-zero. Is this intentional?

Reply:
`CloseAndDie()` does the same thing. I am keeping the same behaviour.